KIPs/kip-7-fungible_token.md

```diff
@@ -47,9 +47,16 @@ If the optional field is not marked, the function must be implemented.
 |[name](#name)|O|function name() public view returns (string)|
 |[symbol](#symbol)|O|function symbol() public view returns (string)|
 |[decimals](#decimals)|O|function decimals() public view returns (uint8)|
-|[mint](#mint)| O |function mint(address _to, uint256 _value) public returns (bool) |
-|[burn](#burn)| O |function burn(uint256 __value) public |
+|[mint](#mint)| O |function mint(address _to, uint256 _value) public onlyMinter returns (bool) |
```

Comment:
What do you think about separating tables for a set of functions that can be implemented as optional?
For example, to implement mintable, we can think of mint, isMinter, addMinter, and renounceMinter as one set.

Reply:
Do you want to separate non-optional and optional functions, or separate only a function group related with mint?

Reply:
I was trying to split a group by function. PTAL. :)

Reply:
I checked the commit you pushed :)
Thank you for changing!!